mne/io/base.py

```diff
@@ -677,57 +685,65 @@ def annotations(self, annotations, emit_warning=True):
         emit_warning : bool
             Whether to emit warnings when limiting or omitting annotations.
         """
+        new_annot = None
         if annotations is not None:
             if not isinstance(annotations, Annotations):
                 raise ValueError('Annotations must be an instance of '
                                  'mne.Annotations. Got %s.' % annotations)
+
+            new_annot = annotations.copy()
             meas_date = _handle_meas_date(self.info['meas_date'])
-            if annotations.orig_time is not None:
-                offset = (annotations.orig_time - meas_date -
+            if meas_date is None:
+                meas_date = 0
+            elif not np.isscalar(meas_date):
+                if len(meas_date) > 1:
+                    meas_date = meas_date[0] + meas_date[1] / 1000000.
```

Comment:
this is a behavior change no?

Reply:
It should not. Its just handleling the datetime. What it could be is that it might not be covered.

Reply:
so it's a bug fix?

did you add a test for this?